Gotcha/SessionEndpoint.swift

```diff
@@ -16,7 +16,8 @@ class SessionEndpoint {
     func login(email_address: String, password: String, onCompletion: @escaping (JSON) -> Void)
     {
         var body: [String: Any] {
-            return ["data": ["attributes": ["email_address": email_address, "password": password]]]
+            return ["data": ["type": "session",
```

Comment:
As I mentioned before, we should move these to constants.  Or even a class variable since we break the endpoints by sessions.

Reply:
Done